parcels/_datasets/__init__.py

```diff
@@ -0,0 +1,46 @@
+"""
+Datasets compatible with Parcels.
+
+This subpackage uses xarray to generate *idealised* structured and unstructured hydrodynamical datasets that are compatible with Parcels. The goals are three-fold:
+
+1. To provide users with documentation for the types of datasets they can expect Parcels to work with.
+2. To supply our tutorials with hydrodynamical datasets.
+3. To offer developers datasets for use in test cases.
+
+Note that this subpackage is part of the private API for Parcels. Users should not rely directly on the functions defined within this module. Instead, if you want to generate your own datasets, copy the functions from this module into your own code.
+
+Developers, note that only idealised datasets that are (a) quick to generate, and (b) only use dependencies already shipped with Parcels, should be added to this subpackage. Real world datasets should be added to the `parcels-data` repository. No data files should be added to this subpackage.
+
+Parcels Dataset Philosophy
+-------------------------
+
+When adding example datasets, there may be a tension between wanting to add a specific example or wanting to add machinery to generate completely arbitrary datasets (e.g., with different grid resolutions, with different ranges, with different datetimes etc.). There are trade-offs to both approaches:
+
+Working with specific hardcoded examples:
```

Comment:
If you call them 'hardcodes examples' here, I would use the same wording in the paragraph above. So that users understand the link between the two